### PR TITLE
Add options to london visit.

### PR DESCRIPTION
**Pull request by sproell**
Please review this, @casey-collab

### CODE REVIEW

File: english/soar/fly-to-london/fly-to-london.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
 
-So you decide to take a break and sit down on Big Ben.
\ No newline at end of file
+So you decide to take a break and sit down on Big Ben.
+
+Just then, you realize that you have forgotten your pint.
+
+Do you:
+
+[Scream?](../scream/scream.md)
```
Comment (casey-collab) [inline english/soar/fly-to-london/fly-to-london.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/soar/fly-to-london/fly-to-london.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+[Do yoga to relax?](../yoga/yoga.md)
```
Comment (casey-collab) [inline english/soar/fly-to-london/fly-to-london.md]: This link is also invalid.

File: english/soar/fly-to-london/fly-to-london.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+[Play minecraft](../you-are-in-minecraft/minecraft.md)
```
Comment (casey-collab) [inline english/soar/fly-to-london/fly-to-london.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by sproell** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :purple_heart:

**Pull request merged**